userguide/tasks.rst

```diff
@@ -811,453 +813,340 @@ Replication Tasks
+.. _zfs_add_replication_task_opts_tab:
 
+.. table:: Replication Task Options
+   :class: longtable
 
-Replication Options
-~~~~~~~~~~~~~~~~~~~
+   +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
+   | Setting                   | Transport | Value          | Description                                                                                                     |
+   |                           |           |                |                                                                                                                 |
+   +===========================+===========+================+=================================================================================================================+
+   | Name                      | ALL       | string         | Enter a descriptive :guilabel:`Name` for the replication.                                                       |
+   +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
+   | Direction                 | ALL       | drop-down menu | Direction of travel. *PUSH* sends snapshots to a destination system. *PULL* receives snapshots from a           |
+   |                           |           |                | destination system. Choosing *PULL* hides the :guilabel:`Periodic Snapshot Tasks` field and renames             |
+   |                           |           |                | :guilabel:`Also Include Naming Schema` to :guilabel:`Naming Schema`.                                            |
+   +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
+   | Transport                 | ALL       | drop-down menu | Method of snapshot transfer:                                                                                    |
+   |                           |           |                |                                                                                                                 |
+   |                           |           |                | * *SSH* is supported by most systems. It requires previously created :ref:`SSH Connections`.                    |
+   |                           |           |                | * *SSH+NETCAT* uses SSH to establish a connection to the destination system, then uses                          |
+   |                           |           |                |   `nc(1) <https://www.freebsd.org/cgi/man.cgi?query=nc>`__ to send an unencrypted data stream for higher        |
+   |                           |           |                |   transfer speeds. This is only an option when replicating to a FreeBSD system that has                         |
```

Comment:
As above: define "only an option" to be more clear.  I suspect it means that the user can choose this, but it only works when there is py-libzfs on the remote end.  Not FreeBSD specific, either, could be FreeNAS or TrueNAS or maybe anything with py-libzfs.

Reply:
The middleware notes about SSH+NETCAT included this caution about FreeBSD systems with py-libzfs. I'll unify this text with the changes I made to the similar text above.